simulation.py

```diff
@@ -0,0 +1,90 @@
+from thousandaire.constants import TRADING_CONFIGS, TRADING_REGIONS
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+COST_FUNCTION = 'cost_function'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays_all = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGIONS if region in raw_data['workdays']}
+    bound_data = {}
+    for region, workdays in workdays_all.items():
+        bound_data[region] = copy.deepcopy(raw_data)
+        for dataset in bound_data[region].values():
+            dataset.set_workdays(workdays)
+    bound_data['workdays'] = workdays_all
+    return bound_data
+
+def extract_data(raw_data, data_list, price_dataset, region):
+    """
+    Extract data on data_list from raw_data.
+
+    Return data will be a dictionary, there will be three keys:
+    (1) 'workdays'
+        Mean the trading region. The type of value would be a DataController.
+    (2) 'target_data'
+        Mean the trading target. The type of value would be a Dataset.
+    (3) 'others'
+        Mean the other data in data_list. The type of value would be a
+        dictionary combined with Dataset.
+    """
+    return {
+        'workdays' : raw_data['workdays'][region],
+        'target_data' : raw_data[region][price_dataset],
+        'others' : {
+            name : raw_data[region][name]
+            for name in data_list if name in raw_data[region]}}
+
+def simulate(alpha_settings_path, un_look):
+    """
+    Act the process of simulating.
+    """
+    try:
+        settings = importlib.import_module(alpha_settings_path).AlphaSettings()
+    except FileNotFoundError as error_1:
+        raise FileNotFoundError("Alpha does not exist.") from error_1
+    except ImportError as error_2:
+        raise ImportError("No available AlphaSettings in %s"
+                          % alpha_settings_path) from error_2
+    data_all = initialize()
+    _, region = settings.target
+    data_required = extract_data(
+        data_all, settings.data_list,
+        TRADING_CONFIGS[settings.target][PRICE_DATASET], region)
+    cost_function = TRADING_CONFIGS[settings.target][COST_FUNCTION](
+        region, data_required['target_data'].keys())
+    simulator = Simulator(settings, data_required, cost_function)
+    # results would put into evaluator after import evaluator.
+    results = simulator.run()
+    if un_look:
+        #This judgement will not dump evaluate result, after import evaluator.
+        pass
+    return results
+
+def main():
+    """
+    Run the simulation process.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("alpha_settings_path")
+    parser.add_argument("--un_look", action='store_true')
+    args = parser.parse_args()
+    simulation_results = simulate(args.alpha_settings_path, args.un_look)
```

Comment:
`un_look` is very confusing for me. Can you change the variable name?
If you think a clear name will be too long for an option, you can use both short options and long options:
https://docs.python.org/3/howto/argparse.html#short-options